src/dotnet/UI.Blazor.App/Pages/HomePage.razor

```diff
@@ -31,9 +39,18 @@
 
     protected override async Task<Model> ComputeState(CancellationToken cancellationToken) {
         var account = await Accounts.Get(Session, CancellationToken.None).ConfigureAwait(false) ?? Account.Guest;
-        var chat = await RecentChatsUI.GetSelectedOrDefaultChat(cancellationToken).ConfigureAwait(false);
-        return new(account, chat?.Id ?? Symbol.Empty);
+        string link = "";
+        if (HistoryUI.IsInitialLocation) {
+            if (BrowserInfo.ScreenSize.Value.IsNarrow())
+                link = Links.ChatPage("");
+            else {
+                var chat = await RecentChatsUI.GetSelectedOrDefaultChat(cancellationToken).ConfigureAwait(false);
```

Comment:
Why it doesn't use `ChatUI.ActiveChat` first?

Reply:
I dont' see property `ChatUI.ActiveChat`. What do you mean?

Reply:
Err, `SelectedChatId`.

Reply:
Got it.